code/datums/diseases/black_goo.dm

```diff
@@ -213,16 +214,16 @@
 
 /datum/language/zombie
 	name = "Zombie"
-	desc = "If you select this from the language screen, expect a ban."
-	color = "zombie"
-
-	speech_verb = "groans"
-	ask_verb = "groans"
-	exclaim_verb = "groans"
-
-	key = "4"
+	desc = "A growling, guttural method of communication, only Zombies seem to be capable of producing these sounds."
+	speech_verb = "growls"
+	ask_verb = "grumbles"
+	exclaim_verb = "snarls"
+	color = "monkey"
+	key = "h"
```

Comment:
> you overrode the language and key with the ones from the monkey language

I replaced the key with h because you can't get monkey and zombie at the same time since being a zombie would wipe out your monkey if you ever were to convert (assuming monkeys even can) and the monkey text looked cooler imo otherwise that's really the only reason